### DISCUSSION TITLE
question about the check box for "Run O and O Shutup"...

### DISCUSSION THREAD

**Discussion by Shurgosa** [ChrisTitusTech/winutil#259] [Q&A] [answered]
Is checking this box, and running the tweak in this tool, the same as setting all of the "recommended" settings in that program?

**Comment by Lakshaya-wq** [comment]
Not at all, O&O Shutup only removes/disables windows spyware. Check out [https://www.oo-software.com/en/shutup10](https://www.oo-software.com/en/shutup10)

**Reply by Shurgosa** [reply]
what I mean is this: I know exactly what O&O shut up does, but I don't know what the check box beside O&O shut in this winutils program does.  What does that check box do?

**Reply by Tech-Ezralow** [reply]
That is a great question,  i would also like to know that answer...

**Comment by Shurgosa** [comment, marked as answer]
> That is a great question, i would also like to know that answer...

@Tech-Ezralow I jumped into Chris Titus' Twitch stream june 23 2023 and asked him and yes as I suspected he confirmed that running the O&O shut up thing from within the winutil he made, simply checks all the recommended flags in O&O shut up and makes those changes to the system.  He also mentioned (if I got this correct. that there is lots of cross-over; many of the recommended things that O&O shut up does, his winutil does as well.  And also he WARNED, correctly, that there are plenty of things that you can turn off in O&O shut up that potentially can cause harm and negative effects, to a persons current OS install and he advised against doing those things....
